nx/test/nx/defn/grad_test.exs

```diff
@@ -1366,6 +1366,47 @@ defmodule Nx.Defn.GradTest do
     end
   end
 
+  describe "outer rule" do
```

Comment:
Brought back by accident too. :) The outer rule is no longer tested because it is no longer a primitive.